app/src/components/UserInfo.js

```diff
@@ -1,11 +1,59 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import { newContextComponents } from "@drizzle/react-components";
 
 const { AccountData, ContractForm } = newContextComponents;
 
+// Function to connect Metamask account to Web3
+//https://metaschool.so/courses/add-metamask-connect-button-to-your-react-app/lesson/decc2d1a-ad8a-48f7-9c29-38a70b654477
+function ConnectMetamask() {
+  const [isWalletInstalled, setIsWalletInstalled] = useState(false);
+  // state for keeping track of current connected account.
+	const [account, setAccount] = useState(null);
+
+  useEffect(() => {
+		if (window.ethereum) {
+			setIsWalletInstalled(true);
+		}
+	}, []);
+
+  async function connectWallet() {
+		  window.ethereum.request({
+				method: "eth_requestAccounts",
+			}).then((accounts) => {
+				setAccount(accounts[0]);
+			}).catch((error) => {
+				alert("Something went wrong");
```

Comment:
Maybe we should alert with the error message, or log it to the console or something just so we can debug.